src/mem/largealloc.h

```diff
@@ -80,22 +81,101 @@ namespace snmalloc
      */
     std::atomic<size_t> available_large_chunks_in_bytes{0};
 
+
+    static constexpr size_t SUPERSLAB_STACK_COUNT = 4;
+
+    /**
+     * Stack of large allocations that have been returned for reuse.

```

Comment:
"Stack of chunks (`Superslab`/`Mediumslab`-sized `Allocslab`s) that have been returned for reuse."?  Add that these all remain committed?